source/fe/fe_poly_tensor.cc

```diff
@@ -572,6 +681,7 @@ FE_PolyTensor<dim, spacedim>::fill_fe_values(
                   for (unsigned int k = 0; k < n_q_points; ++k)
                     for (unsigned int d = 0; d < dim; ++d)
                       output_data.shape_gradients[first + d][k] =
+                        fe_data.dof_sign_change[dof_index] * dof_sign *
```

Comment:
actually, in all of these places :-)

Reply:
By the way, was this here a bug before, since we weren't multiplying with `fe_data.dof_sign_change[dof_index]` as we do in all of the other places where you now multiply with both numbers?

Reply:
I think it was a bug. It was only handled in case of mapping_piola and mapping_nedelec but should be handled in all cases in which 2-forms and 1-forms are mapped